From: Jianyue Wu <wujianyue000@gmail.com>
Subject: mm-rename-swapc-to-folioc-fix
Date: Tue, 4 Aug 2026 19:11:09 +0800

move mm/folio.c to MM CORE

mm/folio.c holds generic folio LRU helpers, not MGLRU-specific
code. Move it to MEMORY MANAGEMENT - CORE, next to
folio-compat.c and folio_batch.h, so get_maintainer.pl routes
reviews to the right people.

Link: https://lore.kernel.org/20260804111109.393127-1-wujianyue000@gmail.com
Fixes: 8ed6d702d442 ("mm: rename swap.c to folio.c")
Signed-off-by: Jianyue Wu <wujianyue000@gmail.com>
Suggested-by: Lorenzo Stoakes <ljs@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/MAINTAINERS~mm-rename-swapc-to-folioc-fix
+++ a/MAINTAINERS
@@ -16988,6 +16988,7 @@ F:	kernel/fork.c
 F:	mm/Kconfig
 F:	mm/debug.c
 F:	mm/folio-compat.c
+F:	mm/folio.c
 F:	mm/highmem.c
 F:	mm/init-mm.c
 F:	mm/internal.h
@@ -17088,7 +17089,6 @@ F:	Documentation/admin-guide/mm/multigen
 F:	Documentation/mm/multigen_lru.rst
 F:	include/linux/mm_inline.h
 F:	include/linux/mmzone.h
-F:	mm/folio.c
 F:	mm/vmscan.c
 F:	mm/workingset.c
 
_
